com.georgefb.haruna.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.georgefb.haruna
+runtime: org.kde.Platform
+runtime-version: 5.14
+sdk: org.kde.Sdk
+command: haruna
+finish-args: 
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=host
+  - --talk-name=ca.desrt.dconf
```

Comment:
```suggestion
```
Is Dconf access really needed?

Reply:
Added because of this https://github.com/flathub/flathub/pull/1062#issuecomment-509096199